lib/edit/offlineFeaturesManager.js

```diff
@@ -2097,7 +2101,7 @@ define([
                  * @returns {*|r}
                  * @private
                  */
-                _makeEditRequest: function(url,adds, updates, deletes, callback, errback) {
+                _makeEditRequest: function(layer,adds, updates, deletes, callback, errback) {
```

Comment:
Minor note: Can you also make sure the documentation [param](https://github.com/Esri/offline-editor-js/blob/master/lib/edit/offlineFeaturesManager.js#L2093) is updated?